cluster/src/main/java/com/hartwig/pipeline/CommonArguments.java

```diff
@@ -39,7 +39,7 @@
 
     String serviceAccountEmail();
 
-    Optional<String> cmek();
```

Comment:
As you pointed out this doesn't make it very external friendly. But... I don't think that's the most important thing right now so we can decide later whether its worth the additional complexity of having a separate flag to disable cmek and make this an optional again.